ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-v1.6-periodics.yaml

```diff
@@ -0,0 +1,76 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
```

Comment:
We pick build01 to try spread the load (1.5 has build02)?

Reply:
This was assigned by this repo's `make jobs`. And the auto-bot sometimes distributes the cluster like  https://github.com/openshift/release/commit/77123db9276168f61ad183208ba458c7027bea23 so let's leave it as it is.